metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -283,4 +280,14 @@ object Embedded {
     new URLClassLoader(urls, classLoader)
   }
 
+  /**
+   * Provides scala3-library dependency for stadalone PC and mdoc instances
+   */
+  def standaloneDependencies(scalaVersion: String): List[Path] = {
+    if (ScalaVersions.isScala3Version(scalaVersion))
+      downloadDependency(scala3Dependency(scalaVersion), scalaVersion)
+    else
+      List.empty
```

Comment:
We might still need to download Scala 2 ones if it's for example `2.13.4`

Reply:
For some reason, everything works well without explicit scala-library in classpath on scala 2 versions.
But nor PC, nor Mdoc doesn't work without it on scala 3.

Anyway, I agree that is better to cover all branches here because it shouldn't affect the PC/Mdoc startup speed. 
Fixed.